README.md

```diff
@@ -3,42 +3,46 @@
 Performance measurement app for KUKSA databroker.
 
 ```
-[00:00:00]  [========================================================================================================]    1000/1000    iterations
-
-Summary:
-  API: SdvDatabrokerV1
-  Elapsed time: 0.16 s
-  Rate limit: None
-  Sent: 1000 iterations * 1 signals = 1000 updates
-  Skipped: 10 updates
-  Received: 990 updates
-  Fastest:   0.042 ms
-  Slowest:   0.686 ms
-  Average:   0.130 ms
+[00:00:10] Group: Frame 1 | Cycle(ms): 10 | Current latency: 0.519 ms [====================================================================]      6/6      seconds
+[00:00:09] Group: Frame 2 | Cycle(ms): 20 | Current latency: 0.562 ms [====================================================================]      6/6      seconds
+[00:00:09] Group: Frame 3 | Cycle(ms): 30 | Current latency: 0.732 ms [====================================================================]      6/6      seconds
+
+Global Summary:
+  API: KuksaValV2
```

Comment:
This should use the same format as the arguments on the command line:
`kuksa.val.v1`, `kuksa.val.v2` etc..